src/yadism/coefficient_functions/heavy/g1_nc.py

```diff
@@ -0,0 +1,194 @@
+import LeProHQ
+import numpy as np
+from scipy.integrate import quad
+
+from ..partonic_channel import RSL
+from . import partonic_channel as pc
+
+
+class GluonVV(pc.NeutralCurrentBase):
+    def NLO(self):
+        """
+        |ref| implements :eqref:`D.3`, :cite:`felix-thesis`.
+        """
+
+        def cg(z, _args):
+            if self.is_below_threshold(z):
+                return 0.0
+            return (
+                self._FHprefactor
+                / z
+                * LeProHQ.cg0("x2g1", "VV", self._xi, self._eta(z))
+            )
+
+        return RSL(cg)
+
+    def NNLO(self):
+        """
+        |ref| implements NLO (heavy) gluon coefficient function, :cite:`felix-thesis`.
+
+        The equation is composed of the coefficient function (here denoted by cg1) :eqref:`D.14` and the scaling coefficient function (here denoted by cgBar1) :eqref:`D.35`.
+
+        The different parts of the coefficient function are as follows:
+
+        The part proportional to ln^2(beta), :eqref:`D.15`.
+        The part proportional to ln(beta), :eqref:`D.16`.
+        The finite 'OK' (non-abelian) part, :eqref:`D.20`.
+        The finite 'QED' (abelian) part, :eqref:`D.26`.
```

Comment:
```suggestion
```
Given that NNLO GluonVV is directly related to the unpolarised heavy, refer directly to :eqref:`D.60` instead.

Reply:
No! eq.60 only holds for light-like coeff's (in my thesis $d$), but only for them and not for $c_g^1$ (for $c_g^0$ they hold "by chance" - i.e. you don't have enough complexity phase space available, if you know what I mean)